src/libraries/System.Private.CoreLib/src/System/Numerics/BitOperations.cs

```diff
@@ -274,6 +280,14 @@ public static int PopCount(ulong value)
                 return (int)Popcnt.X64.PopCount(value);
             }
 
+            if (AdvSimd.IsSupported)
```

Comment:
This should check for `(AdvSimd.Arm64.IsSupported)` since you are using Arm64.AddAcross

Reply:
> AdvSimd.Arm64.IsSupported implies AdvSimd.IsSupported so you need just to check for the former

Ah, I misread your comment to other way round. I will add this check.